ex04.ex

```diff
@@ -55,7 +55,7 @@ defmodule Ex04 do
 
   """
 
-  def min . . . "your code"
+  def min(l), do: reduce(l, &min/2)
```

Comment:
## -3

Sorry, but `min/2` is a built-in function, and you weren't supposed to use them.